scripts/add-fixture.ts

```diff
@@ -92,19 +92,24 @@ const template = {
   console.log();
 
   const packageDir = path.join(fixturesDir, answers.name);
+  const packageJsonPath = path.join(packageDir, 'package.json');
 
   const packageJson = template.packageJson(answers);
   const index = template.index(answers);
 
   await fs.mkdirp(path.join(packageDir, 'src'));
-  await fs.writeJson(path.join(packageDir, 'package.json'), packageJson, {
-    spaces: 2,
-  });
+  await fs.writeJson(packageJsonPath, packageJson, { spaces: 2 });
   await fs.writeFile(path.join(packageDir, 'src/index.ts'), index);
 
-  console.log(packageJson);
-  console.log();
-
+  try {
+    console.log('Trying `bat`...');
+    await run(`bat ${packageJsonPath}`);
```

Comment:
Should this be `cat`, or did you intentionally do [bat](https://github.com/sharkdp/bat)?
```suggestion
    await run(`cat ${packageJsonPath}`);
```

Reply:
It was intentional, so the output is prettier ✨